newIDE/app/src/InstancesEditor/SelectedInstances.js

```diff
@@ -77,12 +81,24 @@ export default class SelectedInstances {
   onRotateEnd: () => void;
   toCanvasCoordinates: (x: number, y: number) => [number, number];
   _screenType: ScreenType;
+  keyboardShortcuts: KeyboardShortcuts;
+  onPanMove: (deltaX: number, deltaY: number, x: number, y: number) => void;
+  onPanEnd: () => void;
 
   pixiContainer = new PIXI.Container();
   rectanglesContainer = new PIXI.Container();
   selectedRectangles = [];
   resizeButtons: { [ResizeGrabbingLocation]: PIXI.Graphics } = {};
   rotateButton = new PIXI.Graphics();
+  /**
+   * When using moving the view with Space key pressed, a click on the buttons
+   * generated by this class won't trigger a resize and interrupt the move of the view.
+   * On the opposite, when resizing an instance, if the Space key is pressed,
+   * the resizing is interrupted to move the view.
+   * This boolean is used to detect the change and call the onResizeEnd action.
+   * Otherwise, the InstancesResizer stays in an intermediate state.
+   */
+  _previousPanMovingGoal: null | 'resizing' | 'viewMoving' = null;
```

Comment:
This can be used for both the resize and the rotate button. 
So this means we don't care about "resizing" really, but more about "is the previous pan goal to move the view or to interact with the button?".

```suggestion
  _previousPanMovingGoal: null | 'buttonInteraction' | 'viewMoving' = null;
```

Reply:
Also, nit: should this be `_currentPanMovingGoal` rather than `_previousPanMovingGoal`? 
When it's used, it's the current thing, even if we update it.

Reply:
True, the fact that current is included in the name make it easier to understand the logic.